src/pages/api/document_submissions/[...path].tsx

```diff
@@ -0,0 +1,26 @@
+import { NextApiHandler } from 'next';
+import { EvidenceApiGateway } from '../../../gateways/evidence-api';
+
+const evidenceApiGateway = new EvidenceApiGateway();
+
+const endpoint: NextApiHandler = async (req, res) => {
+  const query = req.query;
+
+  const { path, currentPage, pageSize, team } = query;
+
+  try {
+    const data = await evidenceApiGateway.getDocumentSubmissionsForResident(
+      req.headers.userEmail?.toString() ?? '',
```

Comment:
`userEmail` header should always contain the email of the staff member when the request is being made by a logged in staff member. The only time that header is not needed is when the request is being made by a resident (because they are not logged in so their email address is not being captured)